openapi.yml

```diff
@@ -74,7 +74,40 @@ paths:
                     type: integer
       parameters:
         - $ref: "#/components/parameters/resourceType"
-
+  /metrics/createdCount/{resourceType}:
+    get:
+      summary: Get the total number of resources created in a specified time period, optionally filtered by group
+      tags:
+        - metrics
+      responses:
+        "200":
+          description: The number of resources created during a specified time period
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
Can you provide a schema for the response?

Reply:
Yes, this was an oversight, thanks.  I added a shared schema for the metrics endpoints (all of which have the same response at the moment) and referenced here.